specification/machinelearning/Azure.AI.ChatProtocol/client.tsp

```diff
@@ -0,0 +1,7 @@
+import "@azure-tools/typespec-client-generator-core";
+
+import "./main.tsp";
+
+using Azure.ClientGenerator.Core;
+
+@@internal(Azure.AI.ChatProtocol.Chat.createStreaming, "csharp");
```

Comment:
`internal` is deprecated.  Use `@@access(<your operation>, Access.internal);` instead

Reply:
updated :)